root/recording/RecordingIndex.js

```diff
@@ -91,6 +94,14 @@ const RecordingAppearancesTable = ({
                   </td>
                   <td>{isolateText(track.name)}</td>
                   <td>{formatTrackLength(track.length)}</td>
+                  {/* The class being added is for usage with userscripts */}
+                  <td className={
+                    track.artistCredit.id === recording.artistCredit.id
+                      ? null
+                      : 'track-recording-clash'}
```

Comment:
Something like `artist-variation` might be more consistent with the `name-variation` class we already have, but it looks fine otherwise.

Reply:
By comparison with name-variation, that would kinda sound to me like "the name is the same but the artists aren't", but maybe not :) @yvanzo, what do you think?

Reply:
Okay, `artist-credit-variation`? :) Mentioning artists somehow would be good though, since it's only indicating if the ACs clash.

Reply:
Perfecto!